src/context.cpp

```diff
@@ -34,11 +34,10 @@
 #include "emitter.hpp"
 
 namespace Sass {
-  using namespace Constants;
-  using namespace File;
-  using namespace Sass;
-  using std::cerr;
-  using std::endl;
+
+  using std::unordered_map;
+  using File::make_canonical_path;
```

Comment:
Bindings are only in the C space, nothing from outside should interface with the C++ internals! There is no stable API on that side!!